interpreter/cling/include/cling/Interpreter/InterpreterCallbacks.h

```diff
@@ -132,6 +132,13 @@ namespace cling {
     ///
     virtual void TransactionCommitted(const Transaction&) {}
 
+    ///\brief This callback is registed at IncrementalParser::Initialize
```

Comment:
What about: "This callback is invoked before a transaction is executed. This event happens after a transaction was committed and LLVM IR was produced."